libraries/chain/wasm_interface.cpp

```diff
@@ -91,6 +91,11 @@ namespace eosio { namespace chain {
       my->apply( code_hash, vm_type, vm_version, context );
    }
 
+   void wasm_interface::do_sync_call( const digest_type& code_hash, const uint8_t& vm_type, const uint8_t& vm_version, apply_context& context ) {
+      ilog("wasm_interface::do_sync_call");
```

Comment:
looks like a debug message left in

Reply:
https://github.com/AntelopeIO/spring/pull/1251/commits/dcfaa50c2e6b429be652c0db47e8dae108ab9dcb